src/Microsoft.Health.Dicom.Api/Controllers/ExtendedQueryTagController.cs

```diff
@@ -138,6 +138,34 @@ public async Task<IActionResult> GetTagAsync(string tagPath)
                 (int)HttpStatusCode.OK, response.ExtendedQueryTag);
         }
 
+        /// <summary>
+        /// Handles requests to get extended query tag errors.
+        /// </summary>
+        /// <param name="tagPath">Path for requested extended query tag.</param>
+        /// <returns>
+        /// Returns Bad Request if given path can't be parsed. Returns Not Found if given path doesn't map to a stored
+        /// error. Returns OK with a JSON body of requested tag error in other cases.
+        /// </returns>
+        [ProducesResponseType(typeof(GetExtendedQueryTagErrorsResponse), (int)HttpStatusCode.OK)]
+        [ProducesResponseType(typeof(string), (int)HttpStatusCode.BadRequest)]
+        [ProducesResponseType(typeof(string), (int)HttpStatusCode.NotFound)]
+        //[Produces("application/json")]
```

Comment:
Any particular reason this is commented out?

Reply:
I thought it made sense to have it, but I didn't see it on other requests